tests/test_secretsmanager/test_secretsmanager.py

```diff
@@ -11,3 +11,12 @@ def test_get_secret_value():
 
     result = conn.get_secret_value(SecretId='java-util-test-password')
     assert result['SecretString'] == 'mysecretstring'
+
+@mock_secretsmanager
+def test_create_secret():
+    conn = boto3.client('secretsmanager', region_name='us-east-1')
+
+    result = conn.create_secret(Name='test-secret', SecretString="foosecret")
+    assert result['ARN'] == (
+        'arn:aws:secretsmanager:us-east-1:1234567890:secret:test-secret-rIjad')
+    assert result['Name'] == 'test-secret'
```

Comment:
Just for sanity's sake, can you add in here getting the secret by name and comparing values.

Also if you dont mind, can you make a test to test getting a secret that doesnt exist?

Reply:
For sure! Will do

Reply:
Thanks for having me add the extra test. Fixed an issue with get_secret_value. Now create and get for secretsmanager work well together 👍